docker/mysql/db/db_upgrade/portal/Update_RMC_DB_2022050300-2022110300.sql

```diff
@@ -0,0 +1,34 @@
+START TRANSACTION;
```

Comment:
Do we want this file in the repo?

Reply:
Yeah we put them here now so they end up in the backend on server.

Reply:
Yeah we put them here now so they end up in the backend on server.